.changes/next-release/bugfix-1c15c377-1231-405a-9047-176e2f6943ce.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Amazon Q: Fix an issue where /dev usage would cause the UI to freeze and take an unusually long time to complete."
```

Comment:
```suggestion
  "description" : "Amazon Q: Fix an issue where /dev usage would cause the UI to freeze and take an unusually long time to complete. (#4269)"
```